public/app/features/plugins/extensions/getPluginExtensions.ts

```diff
@@ -60,24 +62,25 @@ export const getPluginExtensions: GetExtensions = ({ context, extensionPointId,
       // LINK
       if (isPluginExtensionLinkConfig(extensionConfig)) {
         // Run the configure() function with the current context, and apply the ovverides
-        const overrides = getLinkExtensionOverrides(registryItem.pluginId, extensionConfig, frozenContext);
+        const overrides = getLinkExtensionOverrides(pluginId, extensionConfig, frozenContext);
 
         // configure() returned an `undefined` -> hide the extension
         if (extensionConfig.configure && overrides === undefined) {
           continue;
         }
 
+        const path = overrides?.path || extensionConfig.path;
```

Comment:
is there a chance this `path` is an external URL? If so we might not want to add the tracking parameters to it.

Reply:
I don't believe that's possible. I think the extensions link code will validate that a path exists and starts with `/a/${pluginId}`.

Reply:
Yes, it is as @jackw said :point¨